File: aufgabe01/JaNagel/Aufgabe01.py
def fizzBuzz(number):
    for i in range(number):
        yield (str(i) if i%3!=0 and i%5!=0 else ('Fizz' if i%5!=0 else ('Buzz' if i % 3 != 0 else 'FizzBuzz')))
    return

f=fizzBuzz(100)
for i in f:
    print i

File: aufgabe01/cswan/src/aufgabe01.java
public class aufgabe01 {
    public static void main(String[] args) {
        int num = Integer.parseInt(args[0]);

        if(num % 3 == 0)
            System.out.print("Fizz");

        if(num % 5 == 0)
            System.out.println("Buzz");

        System.out.println("\n");
    }
}
